ludwig/api_annotations.py

```diff
@@ -0,0 +1,181 @@
+from typing import Optional
+
+
+def PublicAPI(*args, **kwargs):
+    """Annotation for documenting public APIs. Public APIs are classes and methods exposed to end users of Ludwig.
+
+    If stability="stable", the APIs will remain backwards compatible across minor Ludwig releases
+    (e.g., Ludwig 0.6 -> Ludwig 0.5).
+
+    If stability="experimental", the APIs can be used by advanced users who are tolerant to and expect
+    breaking changes. This will likely be seen in the case of incremental new feature development.
+
+    Args:
+        stability: One of {"stable", "experimental"}
+
+    Examples:
+        >>> from api_annotations import PublicAPI
+        >>> @PublicAPI
+        ... def func1(x):
+        ...     return x
+        >>> @PublicAPI(stability="beta")
```

Comment:
we define the choices above as `{"stable", "experimental"}`, here we mention `"beta"` too

Reply:
Great catch, just fixed!